src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/styles.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+
+import {Color} from 'app/utils/theme';
+import space from 'app/styles/space';
+
+// TODO(style): color #e7eaef and #e7c0bc are not yet in theme
+const LI = styled('li')<{error?: boolean}>`
```

Comment:
Is there a more descriptive name we can use? What about `BreadcrumbItem`?

Also will `error` cause react error warnings?

Reply:
Hm I was not sure about naming this to `BreadcrumbItem` because the parent's name is `Breadcrumb`

Reply:
the `error` doesn't seem to cause any react warning